### PR TITLE
Removed Section formerly 5.D.2.A.4.C, Alternate Housing, which violates RIT housing

### LINKED ISSUE #2: Alternative Housing Clause Does Not Fit Current Practice
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:

The Alternative Housing Clause in 5.D.2.A.4.C states that if an intro member fails their fall evaluation, "the participant is asked to find alternative housing as soon as possible..." which does not align with current practice.

### PR DESCRIPTION
**Pull request by Loerb01**
Check one:
- [X] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

Section formerly 5.D.2.A.4.C, Alternate Housing, is removed to reflect current practice and comply with RIT Housing, which resolves #2.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -994,10 +993,6 @@
 \asubsection{Three-Quarters}
 In a Three-Quarters Vote, a Quorum is reached if the Total Number of Votes Cast is equal to or exceeds three-quarters the Total Number of Possible Votes.
 An option in the vote passes if the number of votes cast for the option equals or exceeds three-quarters of the Total Number of Votes Cast.
-\asubsection{Three Tiered}
```
Comment (rhochgraf21) [inline constitution.tex:997]: Looks like removing Three Tiered voting made it into this PR. Undoing deleting these lines may be a good idea

File: constitution.tex [hunk 2]
```diff
@@ -581,7 +581,6 @@
 \begin{enumerate}
 	\item Introductory Members may be offered Active Membership (\ref{Active Membership}) provided they meet the requirements described in \ref{Expectations of an Introductory Member}, at the discretion of House.
 	\item If an Introductory Member fails to meet the requirements, their membership will be revoked.
-		In addition, the participant is asked to find alternative housing as soon as possible in accordance with all applicable Residence Life policies regarding room changes.
```
Comment (rhochgraf21) [inline constitution.tex:584]: 👍

**Review by rhochgraf21 — CHANGES_REQUESTED** [review]
removing three tiered vote made it into this pr. other than that, looks good

File: constitution.tex (hunk 1 above)
Comment (Sam-Cordry) [reply constitution.tex]: I went ahead and re-added Three Tiered voting to this PR.

**Review by rhochgraf21 — APPROVED** [review]

**Pull request merged**